### PR TITLE
feat: webpack alias tutorial

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

File: tutorials/webpack/alias.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+Aliasing is webpack's handy way to shave time and keystrokes
+off importing frequently used modules. You will need the
+`path` npm module as it is how you will tell webpack where
```
Comment (vkarpov15) [inline tutorials/webpack/alias.md:3]: `path` is a Node.js built-in module, not downloaded via npm: https://nodejs.org/api/path.html

File: tutorials/webpack/alias.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+to look for those specific files. Using the resolve object
+as the parent object and the alias property as the child object,
```
Comment (vkarpov15) [inline tutorials/webpack/alias.md:5]: I'd just say "Using the `resolve.alias` property, you can define aliases for frequently imported modules"

File: tutorials/webpack/alias.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+```
+
+So now when you want to import a file from the library module, you use
+`import {file} from 'Library/fileLocation` or if you include the file
+with the alias, `import {test} from 'Single'`.
+
+# Using alias as a boolean
+
+if for whatever reason you needed webpack to ignore something,
```
Comment (vkarpov15) [inline tutorials/webpack/alias.md:28]: Caps. Also, there's a big reason to use a boolean alias: if you're already loading the library via CDN. Similar to the use case for [Webpack externals](https://masteringjs.io/tutorials/webpack/externals). I don't know if there's a reason to prefer externals over aliases, can you take a look?

**Review by vkarpov15 — CHANGES_REQUESTED** [review]
Couple small comments, but good work overall :+1:

File: tutorials/webpack/alias.md (hunk 3 above)
Comment (IslandRhythms) [reply tutorials/webpack/alias.md]: Took a look, can't find any discussion about the topic so I guess it comes down to preference.

**Pull request merged**